image.go

```diff
@@ -1229,9 +1212,40 @@ func (i *containerImageRef) makeExtraImageContentDiff(includeFooter bool) (_ str
 	return diff.Name(), digester.Digest(), counter.Count, nil
 }
 
+func makeFilteredLayerWriteCloser(wc io.WriteCloser, layerModTime, layerLatestModTime *time.Time) io.WriteCloser {
```

Comment:
Nit: A small comment about this function will be helpful documenting precedence order between `layerModTime` and `layerLatestModTime` ?

Reply:
Added.